test_swagger.py

```
@@ -0,0 +1,63 @@
+                    print(f"   API Title: {data.get('info', {}).get('title')}")
+                    print(f"   API Version: {data.get('info', {}).get('version')}")
+                    print(f"   Total Endpoints: {sum(len(methods) for methods in data.get('paths', {}).values())}")
+                print()
+            else:
+                print(f"❌ {name}: {url}")
+                print(f"   Status: {response.status_code}")
+                print()
+                all_passed = False
+        except requests.exceptions.ConnectionError:
+            print(f"⚠️  {name}: {url}")
+            print(f"   Error: Could not connect. Is the API server running?")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)